src/App.tsx

```diff
@@ -10,9 +10,7 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Outlet />
         </div>
       </div>
     </div>
```

Comment:
lets do some structure refactoring

create 2 folders
1. `pages` : put there `HomePage`, `NotFoundPage`, `PeoplePage`
2. `components`: put all components which we use for our pages